ast.go

```diff
@@ -753,7 +797,7 @@ const (
 
 // String returns the string representation of the node.
 func (node *BinaryExpr) String() string {
-	return fmt.Sprintf("%s %s %s", node.Left.String(), node.Operator, node.Right.String())
+	return fmt.Sprintf("%s%s%s", node.Left.String(), node.Operator, node.Right.String())
```

Comment:
I didn't use the `nodeStringsConcat` function here because handing all of these operators would add quite a bit of complexity to the function and this is the only place where the binary expressions are being handled.